zyte_common_items/pages.py

```diff
@@ -231,33 +234,41 @@ def url(self) -> str:
 
 
 class BaseArticlePage(BasePage, Returns[Article], HasMetadata[ArticleMetadata]):
+    """:class:`BasePage` subclass for :class:`Article`."""
+
     class Processors(BasePage.Processors):
         breadcrumbs = [breadcrumbs_processor]
 
 
 class BaseArticleListPage(
     BasePage, Returns[ArticleList], HasMetadata[ArticleListMetadata]
 ):
+    """:class:`BasePage` subclass for :class:`ArticleList`."""
+
     class Processors(BasePage.Processors):
         breadcrumbs = [breadcrumbs_processor]
 
 
 class BaseArticleNavigationPage(
     BasePage, Returns[ArticleNavigation], HasMetadata[ArticleNavigationMetadata]
 ):
-    pass
+    """:class:`BasePage` subclass for :class:`ArticleNavigation`."""
```

Comment:
Minor: In `components.py`, we added docstrings but left out `pass`. No strong opinion on which one to use but it'd be better to have consistency.